src/components/pricing/PricingThree.astro

```diff
@@ -6,21 +6,21 @@
         <div>
           <span
             class="text-transparent bg-clip-text font-medium tracking-widest bg-gradient-to-r from-cyan-400 via-cyan-500 to-indigo-500 text-xs uppercase"
-            >pricing 3</span
+            >pricing</span
           >
           <h2 class="text-2xl mt-8 font-normal text-white">
             Always know what you’ll pay,
-            <span> transparent pricing for some</span>
+            <span> semitransparent pricing for some</span>
           </h2>
         </div>
       </div>
       <section
         class="flex flex-col rounded-3xl p-[0.060rem] shadow-2xl shadow-vulcan-900 bg-gradient-to-b from-slate-800 via-indigo-500/5">
         <div class="bg-vulcan-900 px-6 sm:px-8 lg:py-8 rounded-3xl h-full">
           <h3 class="mt-5 text-xl text-white">
-            <span class="text-3xl">$6</span> — Hobby
+            <span class="text-3xl">$20</span> — Startup
```

Comment:
Let's also add a limited free plan. Maybe only for personal accounts and with a limited set of cache downloads/uploads per month?